### PR TITLE
4610 upload dual mode

### PR DESCRIPTION
**Pull request by sekmiller**
## New Contributors

Welcome! New contributors should at least glance at [CONTRIBUTING.md](/CONTRIBUTING.md), especially the section on pull requests where we encourage you to reach out to other developers before you start coding. Also, please note that we measure code coverage and prefer you write unit tests. Pull requests can still be reviewed without tests or completion of the checklist outlined below. Thanks!

## Related Issues

- connects to #4610 Scalable Upload Options

## Pull Request Checklist

- [x] Unit [tests][x] none
- [x] Integration [tests][x]
- [x] Deployment requirements:[x]None
- [ ] [Documentation][docs] completed
- [x] Merged latest from "develop" [branch][x] and resolved conflicts

[tests]: http://guides.dataverse.org/en/latest/developers/testing.html
[SQL updates]: https://github.com/IQSS/dataverse/tree/develop/scripts/database/upgrades
[Solr updates]: https://github.com/IQSS/dataverse/blob/develop/conf/solr/7.3.0/schema.xml
[docs]: http://guides.dataverse.org/en/latest/developers/documentation.html
[branch]: http://guides.dataverse.org/en/latest/developers/branching-strategy.html

### CODE REVIEW

File: src/main/java/Bundle.properties [hunk 1]
```diff
@@ -1309,8 +1310,13 @@ dataset.privateurl.noPermToCreate=To create a Private URL you must have the foll
 file.count={0} {0, choice, 0#Files|1#File|2#Files}
 file.count.selected={0} {0, choice, 0#Files Selected|1#File Selected|2#Files Selected}
 file.selectToAddBtn=Select Files to Add
+<<<<<<< HEAD
+file.selectToAdd.tipLimit=File upload limit is {0} bytes per file. 
+file.selectToAdd.tipMoreInformation=Select or drag and drop files from your desktop, directly into the upload widget.
+=======
 file.selectToAdd.tipLimit=File upload limit is {0} per file. 
 file.selectToAdd.tipMoreInformation=For more information about supported file formats, please refer to the <a href="{0}/{1}/user/dataset-management.html#file-handling-uploading" title="File Handling and Uploading - Dataverse User Guide" target="_blank">User Guide</a>.
+>>>>>>> develop
```
Comment (pdurbin) [inline src/main/java/Bundle.properties]: The merge conflicts above need to be resolved.

File: src/main/webapp/editFilesFragment.xhtml [hunk 2]
```diff
@@ -20,8 +20,46 @@
...
                 <!-- Upload -->
-                <ui:fragment id="fileUploadFragment" rendered="#{datasetPage || EditDatafilesPage.showFileUploadFragment()}">
+                <div class="panel-group" role="tablist" aria-multiselectable="true"
+                      id="fileUploadFragment" jsf:rendered="#{datasetPage || EditDatafilesPage.showFileUploadFragment()}">
+                    
+                    <!-- TODO: ACCORDION OPTIONS SCALABLE >1? -->
+                    
+                    <div class="panel panel-default">
+                        
+                        <!-- TODO: New help text needed for these panels, replace placeholder text here and add final msg text to bundle -->
```
Comment (pdurbin) [inline src/main/webapp/editFilesFragment.xhtml]: Is this TODO for now or later? Do we want help text?

File: src/main/java/edu/harvard/iq/dataverse/EditDatafilesPage.java [hunk 3]
```diff
@@ -1671,12 +1693,97 @@ public void uploadStarted() {
...
+
+    private  void setUpRsync() {
+        logger.fine("setUpRsync called...");
+        if (DataCaptureModuleUtil.rsyncSupportEnabled(settingsWrapper.getValueForKey(SettingsServiceBean.Key.UploadMethods))) {
+            try {
+                ScriptRequestResponse scriptRequestResponse = commandEngine.submit(new RequestRsyncScriptCommand(dvRequestService.getDataverseRequest(), dataset));
+                logger.fine("script: " + scriptRequestResponse.getScript());
+                if (scriptRequestResponse.getScript() != null && !scriptRequestResponse.getScript().isEmpty()) {
+                    setRsyncScript(scriptRequestResponse.getScript());
+                    rsyncScriptFilename = "upload-" + workingVersion.getDataset().getIdentifier() + ".bash";
```
[the remark this answers is not in the document]
Comment (matthew-a-dunlap) [reply src/main/java/edu/harvard/iq/dataverse/EditDatafilesPage.java]: Its in DatasetPage as well

File: src/main/java/edu/harvard/iq/dataverse/EditDatafilesPage.java [hunk 4]
```diff
@@ -831,7 +845,10 @@ private String getBundleString(String msgName){
        return ResourceBundle.getBundle("Bundle").getString(msgName);
     }
     
-    
+    public void deleteFilesCompleted(){
```
[the remark this answers is not in the document]
Comment (pdurbin) [reply src/main/java/edu/harvard/iq/dataverse/EditDatafilesPage.java]: It was added in 8dffc46 by @sekmiller and is still used in editFilesFragment.xhtml a couple times around line 340.

[the remark this answers is not in the document]
Comment (pdurbin) [reply src/main/java/edu/harvard/iq/dataverse/EditDatafilesPage.java]: @matthew-a-dunlap in 5733594 I added a comment about how the method is used.

**Comment by pameyer** [comment]
Initial testing in big-data only mode fail to retrieve transfer script, potentially due to reverting the changes made in PR #4951  (DCM version bump).

File: doc/sphinx-guides/source/user/find-use-data.rst [hunk 5]
```diff
@@ -96,19 +95,21 @@ Ingested files can be downloaded in several different ways.
 
 - A subset of the columns of the data
 
+
 .. _rsync_download:
 
-Downloading a Dataverse Package via rsync
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Downloading a Dataverse Package via rsync + SSH
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```
Comment (pameyer) [inline doc/sphinx-guides/source/user/find-use-data.rst]: Uploads are rsync+ssh; downloads are rsync w\out ssh (aka - published package files do not need to be encrypted in transit or use authentication).

**Pull request merged**